.github/scripts/requirements.txt

```diff
@@ -1,2 +1,2 @@
-moz.l10n ~= 0.5.3
-tomli-w ~= 1.0
+moz.l10n ~= 0.6.2
+tomli-w ~= 1.2
```

Comment:
Is there some specific change incldued in tomli-w 1.2 that's relevant to us? `~= 1.0` is equivalent to `>= 1.0, == 1.*`, so this won't change what we're already using.

Reply:
Not really, I tend to update to the latest version when updating other packages, unless there is some compatibility issue.
https://github.com/hukkin/tomli-w/blob/master/CHANGELOG.md